src/gui/sdlmain.cpp

```diff
@@ -1307,6 +1309,40 @@ static SDL_Window* SetWindowMode(const RenderingBackend rendering_backend,
 		if (rendering_backend == RenderingBackend::OpenGl) {
 			flags |= SDL_WINDOW_OPENGL;
 		}
+
+		// We need a context to query the vendor string.
+		const auto temp_window = SDL_CreateWindow(
+		        "", 0, 0, 1, 1, SDL_WINDOW_OPENGL | SDL_WINDOW_HIDDEN);
```

Comment:
We're not showing the temp window anyway, so I think it would be safer to use a realistic window size, like 200 by 200. I can imagine requesting an 1x1 window to perhaphs fail on some OS variants / Window managers.